### PR TITLE
n40: rusty vangers update

### PR DESCRIPTION
**Pull request by kvark**
Part of #1203

### CODE REVIEW

File: content/news/040/index.md [hunk 1]
```diff
@@ -69,6 +69,39 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+that is fast and universal when it comes to viewing angles. It's based
+on an acceleration structure in the form of a voxel octree.
+This work has landed at the start of November, and now it's possible
+to [ride through][vangers-video] the strange worlds
+while looking from behind the car, or even from inside it.
+
+This method runs on all APIs (including OpenGL!), thanks to wgpu/naga
+portability. It's suitably fast, regardless of perspective, even on
+an old macBook with an integrated GPU. It concludes the rendering story
+of the project, and we can shift focus on other areas.
```
Comment (17cupsofcoffee) [inline content/news/040/index.md:95]:
```suggestion
of the project, and the devs can shift focus on other areas.
```

**Review by 17cupsofcoffee — APPROVED** [review]
:+1: LGTM, just going to tweak the last line to be third-person.

**Review by ozkriff — APPROVED** [review]
:fire:

**Comment by ozkriff** [comment]
Thanks!

**Pull request merged**